common/culture/innovations/00_early_modern_innovations.txt

```diff
@@ -0,0 +1,220 @@
+@weapons_and_armor_02 = "gfx/interface/icons/culture_innovations/innovation_weapons_and_armor_02.dds"
+@knight = "gfx/interface/icons/culture_innovations/innovation_knight.dds"
+@hird = "gfx/interface/icons/culture_innovations/innovation_hird.dds"
+@majesty_01 = "gfx/interface/icons/culture_innovations/innovation_majesty_01.dds"
+@majesty_02 = "gfx/interface/icons/culture_innovations/innovation_majesty_02.dds"
+@majesty_03 = "gfx/interface/icons/culture_innovations/innovation_majesty_03.dds"
+@nobility_01 = "gfx/interface/icons/culture_innovations/innovation_nobility_01.dds"
+@nobility_02 = "gfx/interface/icons/culture_innovations/innovation_nobility_02.dds"
+@nobility_03 = "gfx/interface/icons/culture_innovations/innovation_nobility_03.dds"
+@nobility_04 = "gfx/interface/icons/culture_innovations/innovation_nobility_04.dds"
+@misc_inventions = "gfx/interface/icons/culture_innovations/innovation_misc_inventions.dds"
+
+@guns = "gfx/interface/icons/culture_innovations/innovation_gunpowder.dds"
+@production = "gfx/interface/icons/culture_innovations/innovation_production_line.dds"
+@trade = "gfx/interface/icons/culture_innovations/innovation_trade_company.dds"
+@tram = "gfx/interface/icons/culture_innovations/innovation_trams.dds"
+
+#culture_group_military
+innovation_better_seige = {
+	group = culture_group_military
+	culture_era = culture_era_early_modern
+	icon = @siege_weapons
+	
+	modifier = {
+		siege_phase_time = -0.2
+	}
+
+	flag = global_regular
+	flag = early_modern_era_regular
+}
+
+innovation_conscripts = {
+	group = culture_group_military
+	culture_era = culture_era_early_modern
+	icon = @leadership_1
+
+	modifier = {
+		men_at_arms_cap = 1
+		army_maintenance_mult = -0.05
+	}
+
+	flag = global_regular
+	flag = early_modern_era_regular
+}
+
+innovation_defense_charters = {
+	group = culture_group_military
+	culture_era = culture_era_early_modern
+	icon = @fortifications
+
+	modifier = {
+		fort_level = 1
+		garrison_size = 0.05
+	}
+
+	flag = global_regular
+	flag = early_modern_era_regular
+}
+
+innovation_guns = {
+	group = culture_group_military
+	culture_era = culture_era_early_modern
+	icon = @guns
+
+	unlock_maa = riflemen
+
+	flag = global_regular
+	flag = early_modern_era_regular
+}
+
+innovation_professional_army = {
+	group = culture_group_military
+	culture_era = culture_era_early_modern
+	icon = @weapons_and_armor_02
+	
+	modifier = {
+		knight_limit = 1
+	}
+
+	flag = global_regular
+	flag = early_modern_era_regular
+}
+
+innovation_recruitment_centers = {
+	group = culture_group_military
+	culture_era = culture_era_early_modern
+	icon = @levy_building
+
+	modifier = {
+		levy_reinforcement_rate = 0.15
+	}
+
+	flag = global_regular
+	flag = early_modern_era_regular
+}
+
+#culture_group_civic
+innovation_better_building = {
+	group = culture_group_civic
+	culture_era = culture_era_early_modern
+	icon = @civil_construction_2
+
+	modifier = {
+		build_gold_cost = -0.5
+		build_speed = -0.25
```

Comment:
You used wrong modifiers on the docs thus it's OP.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122670473-54721780-d1d3-11eb-9597-3ba1e8016aeb.png)

</details>